spec/controllers/state_file/archived_intake/verification_code_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "rails_helper"
+
+RSpec.describe StateFile::ArchivedIntakes::VerificationCodeController, type: :controller do
+  let(:current_request) { create(:state_file_archived_intake_request, failed_attempts: 0) }
+  let(:email_address) { "test@example.com" }
+  let(:valid_verification_code) { "123456" }
+  let(:invalid_verification_code) { "654321" }
+
+  before do
+    Flipper.enable(:get_your_pdf)
+    allow(controller).to receive(:current_request).and_return(current_request)
+    allow(current_request).to receive(:email_address).and_return(email_address)
+    allow(I18n).to receive(:locale).and_return(:en)
+  end
+
+  describe "GET #edit" do
+    context "when the request is locked" do
+      before do
+        allow(current_request).to receive(:access_locked?).and_return(true)
+      end
+
+      it "redirects to the root path" do
+        get :edit
+
+        expect(response).to redirect_to(root_path)
+      end
+    end
+
+    context "when the request is not locked" do
+      before do
+        allow(current_request).to receive(:access_locked?).and_return(false)
+      end
+
+      it "renders the edit template with a new VerificationCodeForm and queues a job" do
+        expect(ArchivedIntakeEmailVerificationCodeJob).to receive(:perform_later).with(
+          email_address: email_address,
+          locale: :en
+        )
+
+        get :edit
```

Comment:
you can also do this
```suggestion
        expect {
          get :edit
        }.to have_enqueued_job(ArchivedIntakeEmailVerificationCodeJob).with(
          email_address: email_address,
          locale: :en
        )
```